shared/js/ui/models/hamburger-menu.es6.js

```diff
@@ -4,7 +4,7 @@ const parseUserAgent = require('./mixins/parse-user-agent.es6.js')
 function HamburgerMenu (attrs) {
   attrs = attrs || {}
   attrs.tabUrl = ''
-  attrs.isBrowser = this.parseUserAgentString().browser
```

Comment:
It was irking me a bit that a variable called `isBrowser` was not actually a boolean.